src/Build/Evaluation/Expander.cs

```diff
@@ -1947,7 +1947,8 @@ private static class ItemExpander
                 if (itemsOfType.Count == 0)
                 {
                     // .. but only if there isn't a function "Count()", since that will want to return something (zero) for an empty list
```

Comment:
```suggestion
                    // .. but only if there isn't a function "Count", since that will want to return something (zero) for an empty list
                    // or a function "AnyHaveMetadataValue", since that will want to return false for an empty list.
```